modules/base/rendering/renderablesphere.cpp

```diff
@@ -187,6 +209,8 @@ RenderableSphere::RenderableSphere(const ghoul::Dictionary& dictionary)
     , _fadeOutThreshold(FadeOutThresholdInfo, 0.f, 0.f, 1.f, 0.001f)
     , _blendingFuncOption(BlendingOptionInfo)
     , _disableDepth(DisableDepthInfo, false)
+    , _isUsingColorMap(IsUsingColorMapInfo, false)
```

Comment:
For consistency, maybe also rename this variable to active voice
```suggestion
    , _useColorMap(IsUsingColorMapInfo, false)
```